From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Mon, 10 Apr 2023 18:58:57 +0200
Subject: docs: Don't use external intersphinx

Forwarded: Not-Needed
---
 docs/conf.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -14,9 +14,9 @@
 default_role = "py:obj"
 
 intersphinx_mapping = {
-    "python": ("https://python.readthedocs.io/en/latest/", None),
-    "marshmallow": ("https://marshmallow.readthedocs.io/en/latest/", None),
-    "sqlalchemy": ("http://www.sqlalchemy.org/docs/", None),
+    "python": ("/usr/share/doc/python3-doc/html", None),
+    "marshmallow": ("/usr/share/doc/python3-marshmallow/html", None),
+    "sqlalchemy": ("/usr/share/doc/python-sqlalchemy-doc/html", None),
 }
 
 issues_github_path = "marshmallow-code/marshmallow-sqlalchemy"
