diff --git a/tests/test_sqlalchemy_schema.py b/tests/test_sqlalchemy_schema.py
index b9dff39..7439c73 100644
--- a/tests/test_sqlalchemy_schema.py
+++ b/tests/test_sqlalchemy_schema.py
@@ -2,7 +2,7 @@ import marshmallow
 import pytest
 import sqlalchemy as sa
 from marshmallow import Schema, ValidationError, validate
-from pytest_lazy_fixtures import lf
+from pytest import lazy_fixture as lf
 
 from marshmallow_sqlalchemy import SQLAlchemyAutoSchema, SQLAlchemySchema, auto_field
 from marshmallow_sqlalchemy.exceptions import IncorrectSchemaTypeError
